File: pascal/bgi/test/small.pas
program Minimum_Size_Test;
{
 * test program for unit BGI2GRX
}

uses
(*$ifdef __GPC__ *)
  BGI2GRX;
(*$else *)
  graph;
(*$endif*)

var
  grDriver,
  grMode   : Integer;
begin
  grDriver := Detect;
  InitGraph(grDriver, grMode,'');
  CloseGraph;
end.

File: test/bgi/small.c
#include <libbcc.h>

int main(void)
{
  int  gd, gm, err;

  gd = DETECT;
  initgraph(&gd,&gm,"..\\..\\chr");
  err = graphresult();
  closegraph();
  return 0;
}
